### DISCUSSION TITLE
Touch screen support

### DISCUSSION THREAD

**Discussion by OnceI** [ChurchApps/ChurchAppsSupport#315] [FreeShow] [answered]
Would be really nice with touch screen support. Many functions work fine with touch screen, but resizing the various menus is quite difficult now and could be improved.

Btw Nice work so far!

**Comment by vassbo** [comment, marked as answer]
Thanks, I'll see what I can do.

**Comment by vassbo** [comment]
This should be improved in the latest version! Let me know if you have any other specific issues related to this! 😊
